src/app/components/import/ImportModal.js

```diff
@@ -159,7 +159,6 @@ const ImportModal = ({ userKeysList, ...rest }) => {
                 // if (hasCategories(vcardContacts)) {
                 //     return setStep(IMPORT_GROUPS);
                 // }
-                await call();
```

Comment:
Should this be deleted?

Reply:
No, it's there for https://github.com/ProtonMail/proton-contacts/issues/31 . That issue has been unlucky to get the lowest priority...